.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+  1.
+  1.
```

Comment:
WHy three lines?  Or was this meant to increment?

Reply:
I think GH flavored Markdown automatically increments, at least it does in VS Code's preview. Either way I'll make it increment explicitly.